OPUNetGameSelectWnd.cpp

```diff
@@ -860,14 +838,7 @@ void OPUNetGameSelectWnd::OnClickJoin()
 
 void OPUNetGameSelectWnd::OnClickCreate()

```

Comment:
@DanRStevens what do you think about renaming this OnClickCreateGameLobby? Thought it might make it a bit more clear what it does.

Reply:
Not sure about the name. I tend to think of the games list as kind of a lobby. Though there isn't any chat there.

Perhaps the deeper question is if the button labelled "Create" is clear enough.

Reply:
Maybe just leave this alone for now. I think create is fine in context in the GUI itself.